### ISSUE TITLE
swoole_websocket_server::reload后回调函数内无法使用协程

### ISSUE THREAD

**Issue by crazywhalecc** [swoole/swoole-src#2281]
Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a simple script for reproducing the error.
```php
$s = new swoole_websocket_server('0.0.0.0',10086);
$s->on("message", function($server, $frame){
    if($frame->data == "reload") $server->reload();
    elseif($frame->data == "stat") $server->push($frame->fd, "协程uid: ".Co::getuid());
});
```
reload后getuid就成了-1，并且此时协程客户端均无法使用

4. What version of Swoole are you using (show your `php --ri swoole`)?
![image](https://user-images.githubusercontent.com/20330940/50721845-2848a600-1101-11e9-8eec-26e70b70303e.png)

5. What is your machine environment used (including version of kernel & php & gcc) ?
Ubuntu 16.04 LTS, PHP v7.2.13 NTS

**Comment by crazywhalecc** [comment]
![image](https://user-images.githubusercontent.com/20330940/50721873-9db47680-1101-11e9-878e-5f8e49cdf46e.png)

**Comment by twose** [comment]
Fixed in 3ac2337

**Comment by twose** [comment]
更新到v4.2.12
